docs/user-guide/notebooks/Plots.ipynb

```diff
@@ -289,6 +289,35 @@
     "fig = plt.figure(figsize=(10, 8))\n",
     "main_ax_artists, sublot_ax_arists = hist_1.plot_ratio(pdf)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using the `.plot_ratio` API you can also make efficiency plots (where the numerator is a strict subset of the denominator)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hist_3 = hist_2.copy() * 0.7\n",
+    "hist_2.fill(np.random.uniform(-5, 5, 600))\n",
+    "hist_3.fill(np.random.uniform(-5, 5, 200))\n",
```

Comment:
Note: One of the issues with filling with random values it that this will sometimes throw an error:
```
Traceback (most recent call last):
  File "plotting_cli.py", line 122, in <module>
    main_ax_artists, sublot_ax_arists = hist_3.plot_ratio(
  File "/miniconda/envs/test/lib/python3.10/site-packages/hist/basehist.py", line 491, in plot_ratio
    return plot._plot_ratiolike(
  File "/miniconda/envs/test/lib/python3.10/site-packages/hist/plot.py", line 663, in _plot_ratiolike
    ratio_uncert = ratio_uncertainty(
  File "/miniconda/envs/test/lib/python3.10/site-packages/hist/intervals.py", line 163, in ratio_uncertainty
    ratio_uncert = np.abs(clopper_pearson_interval(num, denom) - ratio)
  File "/miniconda/envs/test/lib/python3.10/site-packages/hist/intervals.py", line 105, in clopper_pearson_interval
    raise ValueError(
ValueError: Found numerator larger than denominator while calculating binomial uncertainty
```
Maybe worth adding a warning to this example?